lib/tasks/one_timer.rake

```diff
@@ -9,4 +9,18 @@ namespace :one_timer do
       Teams::UpdateTotalMembersCountService.instance.update_count(team)
     end
   end
+
+  desc 'Generate certificate for completed courses for sayaji learners'
+  task generate_certificate_for_completed_courses: :environment do
+    sayaji = LearningPartner.find(13)
+    certificate_template = sayaji.certificate_templates.find_by(active: true)
+
+    return if certificate_template.blank?
+
+    enrollments = Enrollment.joins(:user).where(course_completed: true,
+                                                users: { learning_partner_id: sayaji.id, role: 'learner' })
+    enrollments.find_each do |enrollment|
+      GenerateCourseCertificateJob.perform_async(enrollment.course_id, enrollment.user_id, certificate_template.id)
```

Comment:
One problem here is that the job won't gets enqueed from local, this has to be trggered from the application.